### PR TITLE
adding nsb computation in data quality from averaged baseline

### PR DESCRIPTION
**Pull request by calispac**
This PR contains 
- more tests files (see #247 )
- NSB rate computation for data_quality.py pipeline
- fixed test_dataquality.py by using more appropriate test files

### CODE REVIEW

**Comment by calispac** [comment]
The plotting fails since the nsb rate compute is NaN since the baseline shift is negative. I think we should change the input files from which the dark baseline is computed. (i.e. real dark file)

**Comment by calispac** [comment]
The input files for the test of data_quality have been changed to more appropriate tests files. Now I discovered that the `.zfits` file that we use for testing were written with 12bits baseline instead og 16bits baseline. Which then screw up completly the baseline shift computation in the test (i.e. negative baseline shift --> nsb_rate = NaN.)

**Comment by calispac** [comment]
@Etienne12345 and @dneise  could we produce a bunch of recent `.zfits` test files (10 to 100 events) for digicampipe?

See #249 and #250

**Comment by calispac** [comment]
Once tests passes it is ready to merge. any comments?

File: digicampipe/io/hdf5.py [hunk 1]
```diff
@@ -23,8 +25,14 @@ def digicamtoy_event_source(
         maximum number of events to read
     camera : utils.Camera() default: utils.DigiCam
     chunk_size : Number of events to load into the memory at once
+    event_id : TODO
     """
```
[the remark this answers is not in the document]
Comment (calispac) [reply digicampipe/io/hdf5.py]: This is a reader from a monte carlo file so I don't really know how to implement this at the moment except to jump to a specific event_number_in_file

[the remark this answers is not in the document]
Comment (yrenier) [reply digicampipe/io/hdf5.py]: you could just skip all event until you reach that one

[the remark this answers is not in the document]
Comment (calispac) [reply digicampipe/io/hdf5.py]: I mean there is a difference between event_id and event_number_in_file.
The Monte Carlo of course does not have an even_id.
But ok we could just go for even_number_in_file = event_id. Although I am not sure it is needed at the moment

**Pull request merged**